torch/_dynamo/variables/torch.py

```diff
@@ -283,7 +283,7 @@ def __init__(self, value, **kwargs):
         self.value = value
 
         assert not isinstance(
-            value, (torch.dtype, torch.device, torch.layout)
+            value, (torch.dtype, torch.device)
```

Comment:
Now layout can be both a torch variable or a ConstantVariable?? Either revert all the layout changes or fix all the issues, but don't leave divergent paths.

Reply:
Yeah let me get rid of making it a constant variable. There are some other paths that want it to be TorchVariable